IFC4x3/Sections/Resource definition data schemas/Schemas/IfcExternalReferenceResource/Entities/IfcDatasetInformation/DocEntity.xml

```diff
@@ -4,7 +4,9 @@
 		<DocAttribute Name="SchemaReference" UniqueId="53e19df0-e203-4029-bdcd-56df644876da" DefinedType="IfcURIReference" AttributeFlags="1">
 			<Documentation>An optional attribute for referencing a schema that defines the possible content and structure of the dataset (e.g. xsd).</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="DatasetInfoForObjects" UniqueId="b516123d-f0a9-4d8d-b03f-d43496183c16" DefinedType="IfcRelAssociatesDataset" AggregationType="3" Inverse="RelatingDataset" />
+		<DocAttribute Name="DatasetInfoForObjects" UniqueId="b516123d-f0a9-4d8d-b03f-d43496183c16" DefinedType="IfcRelAssociatesDataset" AggregationType="3" Inverse="RelatingDataset">
+			<Documentation>The dataset information with which objects are associated.</Documentation>
```

Comment:
This one seems inversed. Should it not be "The dataset for which the information applies."?

Reply:
> This one seems inversed. Should it not be "The dataset for which the information applies."?

I think that I agree, but it is completely in line with the corresponding IfcDocumentInformation.DocumentInfoForObjects so in that case, I think that we should update both.
An alternative description could be "The objects for which the dataset/document information applies."

Reply:
I updated the description in [ceafeae](https://github.com/bSI-InfraRoom/IFC-Specification/pull/869/commits/ceafeaefdfa0f366ca709c7d40619a8a7c4a5da0).
Should I also update DocumentInfoForObjects?

Reply:
Also updated IfcDocumentInformation.DocumentInfoForObjects in [39a07c2](https://github.com/bSI-InfraRoom/IFC-Specification/pull/869/commits/39a07c27fb24abced66e01e611f9230ab1d73764)